homeassistant/components/tado/services.yaml

```diff
@@ -61,3 +61,19 @@ set_climate_temperature_offset:
           max: 10
           step: 0.01
           unit_of_measurement: "°"
+
+add_meter_reading:
+  fields:
+    config_entry:
+      required: true
+      selector:
+        config_entry:
+          integration: tado
+    reading:
+      required: true
+      selector:
+        number:
+          mode: box
+          min: 0
+          step: 1
+          unit_of_measurement: "m³ or kWh"
```

Comment:
Should we omit this if it can be any of them (as we don't know).
Likely looks strange in UI?

Reply:
I understand the concern, as there is usually just one unit of measurement. I thought it looked kind of nice in the UI like this. I quess I could remove it here as the field description already mentions both.

> Reading in m³ or kWh without decimals.